content/ui-patterns/empty-states.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Empty states
+---
+
+import {Flex, Box, Text, BorderBox} from '@primer/components'
+
+Empty states appear when there is no content on a page or feature. The blankslate component is used to represent large empty states across our products. These guidelines demonstrate best practices for using the blankslate component and designing empty states. If you're looking for guidelines on implementation, please refer to [Primer CSS](https://primer.style/css/components/blankslate).
+
+## The blankslate component
+The blankslate is made up of several elements that work together to inform the user about a feature and how to proceed forward. Below are the different elements of the component and how to modify them.
+
+![blankslate anatomy](https://user-images.githubusercontent.com/6846673/62806713-78795380-baa8-11e9-9cbd-1a56fef3247b.png)
+
+### Graphic
+The graphic can bring delight, preview an interface element, or represent the goal of the feature. Graphics should be placed intentionally and with thought about the intention of the content. Graphics also differ in meaning and appeal to the user, which is why the blankslate component has multiple variations. These variations are outlined later on this page.
+
+### Primary Text
+Primary text is the first piece of information in the empty state presented to the user. It can make a user feel much more comfortable to engage with the content or begin a feature flow. Primary text should sound welcoming, human, and convey the intention of the feature.
```

Comment:
The goal of this content should be to help someone make a decision about how to effectively use an empty state, the first 2 sentences don't help me make a decision. I don't think this content is bad, but I do think it could be more helpful. I'd suggest jumping straight into usage guidelines, using command/imperative tense (because this helps give people direction and answers). 

For example: "Use Primary text to make the purpose of the empty state clear, help users feel comfortable about the feature they're engaging with, and the flow that might follow...

Reply:
Noted, thank you!